npc/re/quests/quests_aldebaran.txt

```diff
@@ -0,0 +1,159 @@
+//================= Hercules Script =======================================
+//=       _   _                     _
+//=      | | | |                   | |
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2012-2016  Hercules Dev Team
+//= Ridley
+//=
+//= Hercules is free software: you can redistribute it and/or modify
+//= it under the terms of the GNU General Public License as published by
+//= the Free Software Foundation, either version 3 of the License, or
+//= (at your option) any later version.
+//=
+//= This program is distributed in the hope that it will be useful,
+//= but WITHOUT ANY WARRANTY; without even the implied warranty of
+//= MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//= GNU General Public License for more details.
+//=
+//= You should have received a copy of the GNU General Public License
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Nightmare Clock Tower Dungeon
+//================= Description ===========================================
+//= NPCs related to Aldebaran Clock Tower (Nightmare)
+//================= Current Version =======================================
+//= 1.0
+//=========================================================================
+c_tower1,232,222,5	script	Belljamin Button#ct	4_M_RUSMAN2,{
+	mes("[Belljamin Button]");
+	mes("The monsters here are able to control time.");
+	if (BaseLevel < 150) {
+		mes("Come back when you've reached ^0000FFLevel 150^000000.");
+		close;
+	}
+	mes("Time, memories, environments... all can become warped here.");
+	next;
+	mes("[Belljamin Button]");
+	mes("I'm  begging you not to ask more about it. It might put your life in danger.");
+	next;
+	switch (select("Don't ask", "Go to 2nd floor of warped tower", "Go to 3rd floor of warped tower")) {
+	case 1:
```

Comment:
https://github.com/HerculesWS/Hercules/wiki/Coding-Style
Chapter 1, the one here Ridley used is correct